diff --git a/src/FastViewDX12/MainForm.TransformToolbar.cs b/src/FastViewDX12/MainForm.TransformToolbar.cs
index 0203c1c..c8bcae5 100644
--- a/src/FastViewDX12/MainForm.TransformToolbar.cs
+++ b/src/FastViewDX12/MainForm.TransformToolbar.cs
@@ -1,4 +1,5 @@
 using System;
+using System.ComponentModel;
 using System.Drawing;
 using System.Drawing.Drawing2D;
 using System.Windows.Forms;
@@ -312,8 +313,14 @@ public sealed partial class MainForm
                 false;
         }
 
+        [Browsable(false)]
+        [DesignerSerializationVisibility(
+            DesignerSerializationVisibility.Hidden)]
         public bool Selected { get; set; }
 
+        [Browsable(false)]
+        [DesignerSerializationVisibility(
+            DesignerSerializationVisibility.Hidden)]
         public bool LocalOrientation { get; set; }
 
         protected override void OnMouseEnter(
@@ -714,10 +721,13 @@ public sealed partial class MainForm
                         globeBounds.Height / 2);
             }
 
+            Font baseFont =
+                SystemFonts.MessageBoxFont ??
+                Control.DefaultFont;
+
             using var labelFont =
                 new Font(
-                    SystemFonts.MessageBoxFont ??
-                    Control.DefaultFont,
+                    baseFont.FontFamily,
                     7.0f,
                     FontStyle.Bold,
                     GraphicsUnit.Point);
